### PR TITLE
Meta EIP for Ephemeral Testnet Yolo

### PR DESCRIPTION
**Pull request by MadeofTin**
v1 of the ephnet Yolo will have 2537, the current merged spec for 2315, and implement Option (1) of EIP-2565 ("GQUADDIVISOR is set to ‘20’ per EIP-198. We recommend changing the value of this parameter to ‘200’.”)

If accepted. This meta EIP will be used to track the current State of Yolo and I suggest it included as ACTIVE Status. 

Disclaimer: This is for testing basic infrastructure. It will be nuked. It is not for deploying dapps, nor does it define what will go into mainnet. For information on network upgrades, please follow the relevant meta EIPs and ongoing discussion on Ethereum/pm.

### CODE REVIEW

File: EIPS/eip-2657.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Specification 
+
+Name: Yolo
+ID: `YOLO-v1`
+
+  - EIP 2537
+  - EIP 2315 (current merged Specification)
+  - Option (1) of EIP-2565 
+    - ("GQUADDIVISOR is set to ‘20’ per EIP-198. We recommend changing the value of this parameter to ‘200’.”)
```
Comment (holiman) [inline EIPS/eip-2657.md:25]: I think this is a no-go, from what I heard from @vorot93

Comment (vorot93) [reply EIPS/eip-2657.md:25]: Unfortunately, yes. However, we may implement option 2 of the EIP - see openethereum/openethereum#11728

Comment (MadeofTin) [reply EIPS/eip-2657.md:25]: Can you clarify @vorot93 ?

Should we change it? or remove it?

Comment (vorot93) [reply EIPS/eip-2657.md:25]: I suggest we leave this out for now.

Comment (MadeofTin) [reply EIPS/eip-2657.md:25]: I followed up on this in the ACDs chat. https://gitter.im/ethereum/AllCoreDevs?at=5ec78c7727513a72fba67dd6 

With no strong objections I'll remove for now. Waiting for a bit to give people time to respond.

File: EIPS/eip-2657.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+| Geth         | x      | x    | x      | x       |
+| Nethermind   | x      | x    |        |         |
+| OpenEthereum | x      | x    |        |         |
+| Trinity      |        |      |        |         |
+
+**Signal** -
+Client intends to participate. *(You are on the bus)*
+
+**Spec** -
+Client is statisfied with the proposed specification. *(You agree with the direction)*
```
Comment (edsonayllon) [inline EIPS/eip-2657.md:46]: "Statisfied" is causing the codespell error here

Comment (MadeofTin) [reply EIPS/eip-2657.md]: Good find Thank you!

Comment (MadeofTin) [reply EIPS/eip-2657.md]: updated

**Pull request merged**